src/mockData/mockLayout.js

```diff
@@ -12,4 +12,4 @@ export const mockLayout1 = [
   { name: "snare", icon: "🥁", beats: [false, false, true, false] }
 ];
 
-console.log("mocklayout in file", mockLayout1);
+// console.log("mocklayout in file", mockLayout1);
```

Comment:
Instead of commenting out we should probably remove the comment.

- [ ] remove comment